app/src/main/java/com/aspirebudgetingmobile/aspirebudgeting/interfaces/TransactionCallBack.java

```diff
@@ -0,0 +1,6 @@
+package com.aspirebudgetingmobile.aspirebudgeting.interfaces;
+
+public interface TransactionCallBack {
```

Comment:
`TransactionCallBack` is too vague. Rename class to `AddTransactionCallBack` so it is easy to understand when this callback is triggered.

Reply:
done